conda/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: openalea.phenomenal
-  version: "1.7.1"
+  version: "1.9.2"
```

Comment:
OK for the moment.
But a better solution is to get the version used in the setup.
See other openalea packages (eg. openalea.mtg)